modules/nw-banp-audit-logging-concept.adoc

```diff
@@ -0,0 +1,106 @@
+// Module included in the following assemblies:
+//
+// * networking/openshift_network_security/AdminNetworkPolicy/logging-anp-policy.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="nw-banp-audit-logging-concept_{context}"]
+= BaselineAdminNetworkPolicy audit logging
+
+Audit logging is enabled in the `BaselineAdminNetworkPolicy` CR by annotating an ANP policy with the `k8s.ovn.org/acl-logging` key such as in the following example:
```

Comment:
Should it be by annotating BANP policy here instead of ANP.

Reply:
updated